### PR TITLE
Add MangaSkanlacje.PL to tachiyomi extensions

### PR DESCRIPTION
**Pull request by tomekatomek123456789**
*Add MangaSkanlacje.PL Extension

### CODE REVIEW

File: src/pl/mangaskanlacje/build.gradle [hunk 1]
```diff
@@ -0,0 +1,18 @@
...
+    extName = 'MangaSkanlacje PL'
+    pkgNameSuffix = 'pl.mangaskanlacje'
+    extClass = '.MangaSkanlacjeCMSSources'
+    extVersionCode = 1
+    libVersion = '1.2'
+	containsNsfw = 'true'
+}
+
+dependencies {
+    compileOnly 'com.google.code.gson:gson:2.8.6'
```
Comment (alessandrojean) [inline src/pl/mangaskanlacje/build.gradle:14]: We're in the process of moving from Gson to Kotlinx.serialization.

Please read #7392.

File: src/pl/mangaskanlacje/src/eu/kanade/tachiyomi/extension/pl/mangaskanlacje/Generator.kt [hunk 2]
```diff
@@ -0,0 +1,224 @@
...
+import java.io.PrintWriter
+import java.security.cert.CertificateException
+import java.time.ZonedDateTime
+import java.time.format.DateTimeFormatter
+import java.util.concurrent.TimeUnit
+import javax.net.ssl.SSLContext
+import javax.net.ssl.TrustManager
+import javax.net.ssl.X509TrustManager
+
+class Generator {
```
Comment (alessandrojean) [inline src/pl/mangaskanlacje/src/eu/kanade/tachiyomi/extension/pl/mangaskanlacje/Generator.kt:20]: Why there's a generator if it's not a multi source?

File: build.gradle [hunk 3]
```diff
@@ -7,7 +7,7 @@ buildscript {
         maven { url 'https://plugins.gradle.org/m2/' }
     }
     dependencies {
-        classpath 'com.android.tools.build:gradle:4.2.1'
+        classpath 'com.android.tools.build:gradle:7.0.0'
```
Comment (alessandrojean) [inline build.gradle:10]: Better to undo this change and keep the same Gradle version.

File: gradle/wrapper/gradle-wrapper.properties [hunk 4]
```diff
@@ -1,5 +1,5 @@
 distributionBase=GRADLE_USER_HOME
 distributionPath=wrapper/dists
-distributionUrl=https\://services.gradle.org/distributions/gradle-7.0-bin.zip
+distributionUrl=https\://services.gradle.org/distributions/gradle-7.0.2-bin.zip
```
Comment (alessandrojean) [inline gradle/wrapper/gradle-wrapper.properties:3]: Same here.

File: src/all/batoto/release/output-metadata.json [hunk 5]
```diff
@@ -0,0 +1,20 @@
+{
```
Comment (alessandrojean) [inline src/all/batoto/release/output-metadata.json:1]: Can remove this file.

**Review by alessandrojean — CHANGES_REQUESTED** [review]

**Comment by alessandrojean** [comment]
Also, this website uses MMRCMS, so you should only edit the MMRCMS multisrc stuff instead of creating an separated extension manually.

You can look at the mmrcms folder inside multisrc to get a better reference of how to do it.

**Comment by arkon** [comment]
Seems like an abandoned PR, so I'm going to close this. Please open another PR if you decide to return to this.

**Pull request closed without merging**